src/libs/workspace-utils.ts

```diff
@@ -61,6 +63,7 @@ export interface AzureContext {
 
 export interface AzureWorkspace extends BaseWorkspace {
   azureContext: AzureContext;
+  policies?: { name: string; namespace: string; additionalData: { [key: string]: string } }[];
```

Comment:
Why does policies contain name and namespace?

Reply:
Oh wait… those are different from billing project name and workspace namespace….